src/utils/isStandardSyntaxUrl.js

```diff
@@ -1,21 +1,35 @@
-import { hasInterpolation } from "../utils"
+import {
+  hasLessInterpolation,
+  hasScssInterpolation,
+} from "../utils"
 /**
  * Check whether a URL is standard
  *
  * @param {string} url
  * @return {boolean} If `true`, the url is standard
  */
 export default function (url) {
-  // SCSS or Less interpolation
-  if (hasInterpolation(url)) { return false }
+  if (url.length === 0) { return true }
 
-  // Inside `'` and `"` work only interpolation
+  // Sass interpolation works anywhere
+  if (hasScssInterpolation(url)) { return false }
+
+  // Inside `'` and `"` work only LESS interpolation
   if (url[0] === "'" && url[url.length - 1] === "'"
     || url[0] === "\"" && url[url.length - 1] === "\""
-  ) { return true }
+  ) {
+    if (hasLessInterpolation(url)) { return false }
+
+    return true
+  }
+
```

Comment:
What if we have postcss simple vars interpolation? Don't we need to return `false` for that also?